fe/fe-core/src/main/java/com/starrocks/persist/OperationType.java

```diff
@@ -92,6 +92,7 @@ public class OperationType {
     // load job for only hadoop load
     public static final short OP_EXPORT_CREATE = 36;
     public static final short OP_EXPORT_UPDATE_STATE = 37;
+    public static final short OP_EXPORT_UPDATE_INFO = 38;
```

Comment:
Adding a new operationType, the roll back need to take the compatibility into consideration.